Portfolio/index.html

```diff
@@ -262,153 +261,448 @@
 }
-.project-text2{
-  display: contents;
-    /* border: 2px solid red; */
-    width: 90vw;
-    line-height: 20px;
-    float: left;
-    margin-top: 90px;
-    margin-left: 500px;
-}
-.project-text2 h1, b{
-    margin: 90px;
-width: 100vw;
-padding: 2px 34px;
-    display: contents;
-}
-.project3{
-    float: left;
-    position:static;
-    display: flex;
+
+.phnbtn button{
+    padding: 22px 63px;
+    margin-right:-12px;
+    width: 232px;
+}
+.phnbtn strong{
+    font-size: 0.8rem;
+}
+
+ 
+
+.gettoknowme{
+   position: relative;
+   display: inline-block;
+   margin-right: 50%;
+    justify-content: center;
+    align-items: center;
+    height: 138vh;
+    width: 404%;
 }
-.project3 img{
-    float: left;
+
+.gettoknowme p{
+    width: 54%;
+    justify-content: center;
+    align-items: center;
+    margin-top: 97px;
+    font-size: 3rem;
+    margin-left: 429px;
+}
+.name{
+    font-size: 3rem;
+        margin-left: 430px;
     
 }
+.gettoknowme h1{
+    margin-left: 577px;
+    width: 585px;
+    font-size: 5rem;
+    align-self: center;
+}
 
+.project-section{
+    margin-right: -1284px;
+}
 
-.project-text3{
+.containerss{
+    /* flex-wrap: wrap; */
+    justify-content: center;
+    align-items: center;
+    /* margin-left: 100px; */
+}
 
-    line-height: 20px;
-    margin-top: 90px;
-    margin-right: 119px;
-    float: right;
-}   
+.myskills{
+    margin-left: 838px;
+    background-color: beige;
+    
+}
 
-.project4{
-    float: right;
 
-    margin-top: 30px;
-    margin-right: 20px;
+.myskills h1 {
+    font-size: 5rem;
+    justify-content: center;
+    align-items: center;
+    margin-left: -171px;
+}
+.image-container img {
+	width: 50px;
+    filter: invert(60%);
+	height: auto;
+}
+.image-container h2 {
+font-size: 0.5rem;
 }
-.project4 img{
-    float: right;
 
-    margin-top: 30px;
-    margin-right: 120px;
+.John{
+    max-width: 100%;
+}
+.abtme{
+    width: 403vw;
+    height: 800px;
+}
+
+.abtme img{
+    height: 79%;
+    width: 48%;
 
 }
 
-.project-text4{
-    display: contents;
-    border: 2px solid red;
-    width: 90vw;
-    line-height: 20px;
-    float: left;
-    margin-top: 90px;
-    margin-left: 500px;
+.contact{
+    padding: 20px 13px;
+     width: 400%;
 }
 
-.project-text4 h1, b{
-    margin: 90px;
-    width: 100vw;
-    padding: 2px 34px;
-        display: contents;
+
+.footer {
+   
+    margin-left: 246px;
+    margin-left: -1px
 }
+.footer li{
+    margin-left: 162px;
+}
+.footer li a{
+    font-size: 1rem;
 
+}
 
 
+.text-right {
+    max-width: 63%;
+    margin-right: 98px;
+    margin-top: -23px;
+
+  }
+  .text-right strong{
+    font-size: 5rem;
+    color: #00D8FF;
+  }
+  .text-right p{
+    color: #9ABEE980;
+    font-size: 3rem;
+  }
+  .text-right b{
+    color: #9ABEE9;
+    font-size: 2rem;
+  }
+  .text-right h1{
+  
+      margin-right: 34px;
+  
+  }
+  
+  .text-left{
+    max-width: 63%;
+    margin-left: -75px;
+    margin-top: 6px;
+  }
+  .text-left p{

```

Comment:
The closing '</body>' tag should be accompanied by a closing '</html>' tag to ensure proper document structure.